snapcraft/tests/test_yaml.py

```diff
@@ -283,7 +283,89 @@ def test_invalid_yaml_invalid_name_as_number(self, mock_loadPlugin):
             snapcraft.yaml.Config()
 
         self.assertEqual(raised.exception.message,
-                         '1 is not of type \'string\'')
+                         'The \'name\' property does not match the required '
+                         'schema: 1 is not of type \'string\'')
```

Comment:
Yeah, but jsonschema's error messages use single-quotes, so I had to bite the bullet and maintain consistency.